custom/conf/app.example.ini

```diff
@@ -1499,39 +1499,51 @@ ROUTER = console
 ;; Prefix displayed before subject in mail
 ;SUBJECT_PREFIX =
 ;;
-;; Mail server
-;; Gmail: smtp.gmail.com:587
-;; QQ: smtp.qq.com:465
-;; As per RFC 8314 using Implicit TLS/SMTPS on port 465 (if supported) is recommended,
-;; otherwise STARTTLS on port 587 should be used.
-;HOST =
-;;
-;; Disable HELO operation when hostnames are different.
-;DISABLE_HELO =
-;;
-;; Custom hostname for HELO operation, if no value is provided, one is retrieved from system.
+;; NOTICE: many options are new in 1.18. For old Gitea, please refer to old app.example.ini
+;; eg: https://github.com/go-gitea/gitea/blob/release/v1.17/custom/conf/app.example.ini
+;;
+;; Mail server protocol. One of "smtp", "smtps", "smtp+startls", "smtp+unix", "sendmail", "dummy".
+;; - sendmail: use the operating system's `sendmail` command instead of SMTP. This is common on Linux systems.
+;; - dummy: send email messages to the log as a testing phase.
+;; If your provider does not explicitly say which protocol it uses but does provide a port,
+;; you can set SMTP_PORT instead and this will be inferred.
+;; Since 1.18
```

Comment:
Yes interesting sentence. This need some extra words I assume? CC @wxiaoguang https://github.com/go-gitea/gitea/commit/466d6ba3f576ab8b04aa33f6eee488c1131de9f5

Reply:
Yes, it was not clear. clarfonthey has updated them, it looks good to me.

Do you have more ideas about how to improve?